superbench/benchmarks/docker_benchmarks/docker_base.py

```diff
@@ -36,24 +44,87 @@ def _preprocess(self):
         Return:
             True if _preprocess() succeed.
         """
-        return super()._preprocess()
+        if not super()._preprocess():
+            return False
+
+        if self._image is None:
+            self._result.set_return_code(ReturnCode.DOCKERBENCHMARK_IMAGE_NOT_SET)
+            logger.error('The image url is not set - benchmark: {}.'.format(self._name))
+            return False
+
+        if self._container is None:
+            self._result.set_return_code(ReturnCode.DOCKERBENCHMARK_CONTAINER_NOT_SET)
+            logger.error('The container name is not set - benchmark: {}.'.format(self._name))
+            return False
+
+        output = run_command('docker pull {}'.format(self._image))
```

Comment:
add `--quiet`? or the progress bar will mess up output

Reply:
done